src/System.Net.Security/src/System/Net/Security/SslClientAuthenticationOptions.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Security
+{
+    public class SslClientAuthenticationOptions
+    {
+        private string _targetHost;
+        private X509CertificateCollection _clientCertificates;
+        private EncryptionPolicy _encryptionPolicy = EncryptionPolicy.RequireEncryption;
+        private X509RevocationMode _checkCertificateRevocation = X509RevocationMode.NoCheck;
+        private SslProtocols _enabledSslProtocols = SecurityProtocol.SystemDefaultSecurityProtocols;
+
+        internal RemoteCertValidationCallback _certValidationDelegate;
+        internal LocalCertSelectionCallback _certSelectionDelegate;
+
+        public bool AllowRenegotiation { get; set; }
+
+        public LocalCertificateSelectionCallback LocalCertificateSelectionCallback { get; set; }
+
+        public RemoteCertificateValidationCallback RemoteCertificateValidationCallback { get; set; }
+
+        public IList<SslApplicationProtocol> ApplicationProtocols { get; set; }
```

Comment:
Did this make it through API review?  I'm pretty sure we have the position of properties being concrete collections, not interfaced collections.  (@KrzysztofCwalina?)

Reply:
Yes, it went throught api review.